builtin/logical/cassandra/path_roles.go

```diff
@@ -123,11 +128,14 @@ func (b *backend) pathRoleCreate(
 		return logical.ErrorResponse(fmt.Sprintf(
 			"Error parsing lease value of %s: %s", leaseRaw, err)), nil
 	}
+	
+	consistencyStr := data.Get("consistency").(string)	
```

Comment:
You could send it through `gocql.ParseConsistencyWrapper` as you are at creds issue time. You can throw away the result but just verify that you don't get an error.